### DISCUSSION TITLE
Nemo: Create Symlinks

### DISCUSSION THREAD

**Discussion by trymeouteh** [linuxmint/discussions#408] [Ideas]
Please add the ability to right click on a file or folder and click copy or by selecting the file or folder and pressing CTRL+C and then right clicking in another directory and select the options "Create Symbolic Link" which will create a symlink of the item you have copied.

Will be much easier creating symlinks this way then having to use the terminal.

**Comment by mtwebster** [comment]
There are a couple alternatives to using the terminal you can try:
- From the context menu: open Nemo's preferences (edit->Preferences) and go to the Context Menus page. Under the 'Selection' section, check 'Make Link'. This will allow you to right-click an item and create a symbolic link in the current folder, which you can then move/cut/paste wherever.
- Using drag-and-drop: Start dragging an item, then hold Alt. The pointer should end up with a ? symbol on it. Once you drop the item, a popup menu will appear allowing you to create a link, move, or copy the item. As a shortcut for a link, begin dragging, then hold Ctrl+Shift simultaneously. The pointer should display a 'link' symbol. Dropping into another location will create a symbolic link back to that item.

**Reply by trymeouteh** [reply]
Then first option will do the trick!
